frontend/src/components/MeetingRoom/LayoutButton/LayoutButton.tsx

```diff
@@ -4,20 +4,25 @@ import WindowIcon from '@mui/icons-material/Window';
 import { ReactElement } from 'react';
 import useSessionContext from '../../../hooks/useSessionContext';
 import ToolbarButton from '../ToolbarButton';
-import useIsSmallViewport from '../../../hooks/useIsSmallViewport';
 
-type LayoutButtonProps = {
+export type LayoutButtonProps = {
   isScreenSharePresent: boolean;
+  isOverflowButton?: boolean;
 };
 
 /**
  * LayoutButton Component
  *
  * Displays a button to toggle the meeting room layout for the user between `grid` and `active-speaker`.
- * @param {boolean} isScreenSharePresent - Indicates whether there is a screenshare currently in the session.
+ * @param {LayoutButtonProps} props - the props for the component
+ *  @property {boolean} isScreenSharePresent - Indicates whether there is a screenshare currently in the session.
+ *  @property {boolean} isOverflowButton - whether the button is in the ToolbarOverflowMenu
```

Comment:
same here please with the `(optional)`, also good job re-writing the jsdoc to better match other components 

I'll stop asking for the rest of the `isOverflowButton` jsdocs 😂